src/AppBundle/Utils/Slugger.php

```diff
@@ -27,6 +27,6 @@ class Slugger
      */
     public function slugify($string)
     {
-        return trim(preg_replace('/[^a-z0-9]+/', '-', strtolower(strip_tags($string))), '-');
+        return preg_replace('/\s+/', '-', mb_strtolower(trim(strip_tags($string))));
```

Comment:
note that this is very likely, as the default one was *not* UTF-8 on PHP 5 (PHP 7 changed the default encoding to UTF-8)